src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form>
+      <input
+        type="text"
+        class="search-bar search-bar--big"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        class="search-bar search-bar--small"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
The form currently lacks a submit button. If the form is intended to be functional, consider adding a button element with type="submit" to allow users to submit the form.